modules/kubernetes-addons/crossplane/main.tf

```diff
@@ -0,0 +1,6 @@
+module "helm_addon" {
```

Comment:
I think you need to add the "short license" file header to these files. @jomcy-amzn ?

Reply:
No I don't think we need them. Repos like https://github.com/aws-ia/terraform-aws-control_tower_account_factory have not included license files in the source files. Root level is good enough.

Reply:
ok great!